docs/tutorials/simple-date-clock-application.adoc

```diff
@@ -0,0 +1,435 @@
+= A simple date-clock application: tutorial cloud-harness
+
+In this small tutorial, we will see different aspects about the developement of applications with cloud harness through the developement from scratch of a small webapp that fetch information from a server on a regular basis.
```

Comment:
usually cloud harness is spelled CloudHarness, but I'm sure you'll find a lot of inconsistencies

Reply:
typo: fetch->fetches